<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Doc&amp;kWidget</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centerWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="inboxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="inboxHeaderLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(224, 255, 248);
font: 16pt &quot;Times New Roman&quot;;
color: rgb(107, 107, 107);</string>
       </property>
       <property name="text">
        <string>You're chatting with ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="inboxBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="senderLayout">
     <item>
      <widget class="QLineEdit" name="messageEdit">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>60</number>
       </property>
       <property name="placeholderText">
        <string>Hello, who is there?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="messageLengthLabel">
       <property name="text">
        <string>00/80</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileButton">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="connectionLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="connHeaderLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(219, 255, 234);
font: italic 16pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>All connections: ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="connTable">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashLine</enum>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>80</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Gallon</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Username</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ping</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Gallon</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Client</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1341</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="headerLayout" stretch="1,1,1,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QFormLayout" name="statusLayout">
       <property name="formAlignment">
        <set>Qt::AlignCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="serverConnLabel">
         <property name="text">
          <string>Server connections:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="serverConnEdit">
         <property name="text">
          <string>5</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="clientConnLabel">
         <property name="text">
          <string>Client connections:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="clientConnEdit">
         <property name="text">
          <string>10</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="infoLayout">
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ipAddressEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Your name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameEdit">
         <property name="text">
          <string>Gallon</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ipAddressLabel">
         <property name="text">
          <string>Your IP address:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="connectLayout">
       <item>
        <widget class="QPushButton" name="scanButton">
         <property name="text">
          <string>Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectToButton">
         <property name="text">
          <string>Connect to</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="aboutAndLogLayout">
       <item>
        <widget class="QPushButton" name="aboutButton">
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logButton">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>241</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="fileTransferLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="fileTransferHeaderLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 254, 210);
color: rgb(121, 121, 121);
font: italic 16pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>File transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="fileTransferList">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="FileTransferProgress">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
